workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt

```diff
@@ -93,6 +102,37 @@ class TokenSelectionTests : MockNetworkTest(numberOfNodes = 4) {
         // TODO: Assert something...
     }
 
+    fun <T : TokenType> issuerCriteria(token: T, issuer: Party): QueryCriteria {
```

Comment:
We already have issuer criteria helper function `tokenAmountWithIssuerCriteria` and ownedTokensByTokenIssuer

Reply:
whoops thanks using that now.....